accounts/abi/bind/backends/simulated.go

```diff
@@ -525,13 +525,13 @@ func (b *SimulatedBackend) SuggestGasPrice(ctx context.Context) (*big.Int, error
 	if b.pendingBlock.Header().BaseFee != nil {
 		return b.pendingBlock.Header().BaseFee, nil
 	}
-	return big.NewInt(1), nil
+	return common.Big1, nil
```

Comment:
Unassuming calling-code could modify this returned value and clobber the value of the constant.  Should return a copy of `common.Big1`.